app/src/main/scala/org/alephium/explorer/web/DocumentationServer.scala

```diff
@@ -16,14 +16,18 @@
 
 package org.alephium.explorer.web
 
+import scala.concurrent.{ExecutionContext, Future}
+
 import akka.http.scaladsl.server.Route
-import sttp.tapir.swagger.akkahttp.SwaggerAkka
+import sttp.tapir.swagger.{SwaggerUI, SwaggerUIOptions}
 
 import org.alephium.api.OpenAPIWriters.openApiJson
 import org.alephium.explorer.docs.Documentation
 
-object DocumentationServer extends Server with Documentation {
-
-  val route: Route =
-    new SwaggerAkka(openApiJson(docs, dropAuth = false), yamlName = "explorer-backend-openapi.json").routes
+class DocumentationServer()(implicit val executionContext: ExecutionContext)
+    extends Server
+    with Documentation {
+  val route: Route = toRoute(
+    SwaggerUI[Future](openApiJson(docs, dropAuth             = false),
```

Comment:
don't forget to test this manually. We had an issue for full node

Reply:
just tested, all good :+1: